src/apis/modTools.es6.js

```diff
@@ -0,0 +1,25 @@
+import apiRequest from '../apiBase/apiRequest';
+
+const parsePostBody = apiResponse => {
+  const { body } = apiResponse.response;
+
+  return apiResponse;
+};
+
+const remove = (apiOptions, fullname, spam) => {
+    // Remove a link, comment, or modmail message.
+    const body = {
+    	id: fullname,
+    	spam: spam
+    };
+
+    return apiRequest(apiOptions, 'POST', 'api/remove', { body, type: 'form' }).then(parsePostBody);
+}
+
+const approve = (apiOptions, fullname) => {
+    // Approve a link or comment
+    const body = { id: fullname };
```

Comment:
Kept this `body` const now that we're passing fullname (it's called id on the server).